### PR TITLE
Readme on target dir var, minor edits for the new 'yes' flag

### PR DESCRIPTION
**Pull request by zachpeterpaul**
Added to readme that the target dir var can only be set in .pcconfig or an env var, but not in pcconfig.json (which itself cannot be found without knowing the target dir). This should likely resolve https://github.com/playcanvas/playcanvas-sync/issues/20
Minor readme and code edits for the new "yes" flag for push/pullAll

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +53,8 @@ switch to the browser code editor.
   parseIgnore                 list assets matched by pcignore.txt
 ```
 
-To automatically answer "yes" to any prompts that might print
-on the command line from `pushAll` or `pullAll` commands, simply supply `-y` or `--yes` arguments:
-
-```
-pcsync pushAll -y
-```
```
Comment (yaustar) [inline README.md:61]: Any reason to remove the CLI example?

File: README.md [hunk 2]
```diff
@@ -246,6 +242,9 @@ All listed key-value pairs are necessary. You can keep
 some of them in `.pcconfig` in your home directory,
 and others in `pcconfig.json` in your project.
 
+To find `pcconfig.json`, `pcsync` needs to know your target directory,
+so `PLAYCANVAS_TARGET_DIR` can only be set in `.pcconfig` or an environment variable.
+
```
Comment (yaustar) [inline README.md:247]: I would phase this:
```
All listed key-value pairs are necessary. 

`PLAYCANVAS_TARGET_DIR` must be in `.pcconfig` in the home directory or an environment variable.

The reminder can be in `pcconfig.json` in your project target directory or all key-value pairs can be in `.pcconfig` in the home directory.
```

**Review by yaustar — APPROVED** [review]
Approved with comments

File: README.md (hunk 1 above)
Comment (zachpeterpaul) [reply README.md:61]: I felt it made that paragraph longer than necessary

File: README.md (hunk 2 above)
Comment (zachpeterpaul) [reply README.md]: I've merged what you suggest with what I had. Please feel free to suggest more readme edits if necessary, probably in a separate pr.

**Pull request merged**